### PR TITLE
Fix for MetOffice reprocess_all_met

### LINKED ISSUE #138: The reprocess_all_met needs to be fixed with the get_latest_revision pointing to the NRL job

### PR DESCRIPTION
**Pull request by fabiolrdiniz**
## Description
This PR is fixing an issue that has been introduced by #133.

### Issue(s) addressed
- fixes #138

## Dependencies
None

## Impact
None

### CODE REVIEW

File: python/test/reprocess_all_met.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def submit_job_for_date(date_str):
 
     batch = boto3.client('batch')
 
-    latest = get_latest_revision('ios_ingest_met_job')
+    latest = get_latest_revision('ios_ingest_nrl_job')
     batch.submit_job(
         jobName='met_%s' % date_str,
         jobDefinition='ios_ingest_met_job:%d' % latest,
```
Comment (hahnd) [inline python/test/reprocess_all_met.py]: This needs to change too.

File: python/test/reprocess_all_met.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def submit_download_job_for_date(date):
 
     # submit the job
     batch = boto3.client('batch')
-    latest = get_latest_revision('ios_ingest_met_job')
+    latest = get_latest_revision('ios_ingest_nrl_job')
     batch.submit_job(
         jobName='met_%s' % date,
         jobDefinition='ios_ingest_met_job:%d' % latest,
```
Comment (hahnd) [inline python/test/reprocess_all_met.py]: This needs to change too.

Maybe create a variable, jobDefinitionName = 'ios_ingest_nrl_job' and use it in those four places.

**Review by hahnd — CHANGES_REQUESTED** [review]

File: python/test/reprocess_all_met.py (hunk 1 above)
Comment (fabiolrdiniz) [reply python/test/reprocess_all_met.py]: Thanks, @hahnd.

File: python/test/reprocess_all_met.py (hunk 2 above)
Comment (fabiolrdiniz) [reply python/test/reprocess_all_met.py]: Thanks, @hahnd.

**Comment by fabiolrdiniz** [comment]
Hi, @hahnd. Thanks for looking into this. The variable `jobDefinitionName` has been added in the last commit.

**Review by hahnd — APPROVED** [review]
Looks good!

**Pull request merged**